### PR TITLE
UGENE-7661 Crash filtering and selection in project + reopening file

### PR DESCRIPTION
**Pull request by DmitriiSukhomlinov**
https://ugene.dev/tracker/browse/UGENE-7661

### CODE REVIEW

**Comment by DmitriiSukhomlinov** [comment]
I re-run GUI tests, but failure on the last run wasn't related to this feature and, I spose, absolutely random

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp [hunk 1]
```diff
@@ -3108,6 +3108,47 @@ GUI_TEST_CLASS_DEFINITION(test_7659) {
...
+     
+    // Close the chrM tab.
+    GTMenu::clickMainMenuItem(os, {"Actions", "Close active view"}, GTGlobals::UseKey);
+
+    // Rename the file in the storage from "chrM.sorted.bam.ugenedb" to "Renamed.ugenedb".
+    // UGENE displays the message "File Modification Detected".Click OK.
+    GTUtilsDialog::waitForDialog(os, new MessageBoxDialogFiller(os, "OK", "was removed"));
+    QFile f(sandBoxDir + "/chrM.sorted.bam.ugenedb");
+    f.rename(sandBoxDir + "/Renamed.ugenedb");
+    GTGlobals::sleep(8000);
```
Comment (yalgaer) [inline src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp]: Use GTUtilsDialog::checkNoActiveWaiters, so you will wait the minimum required time

**Review by yalgaer — CHANGES_REQUESTED** [review]
Looks good. The only requested change is about sleep() method in the test

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp (hunk 1 above)
Comment (DmitriiSukhomlinov) [reply src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp]: OK

**Comment by DmitriiSukhomlinov** [comment]
wait, some problems

**Comment by DmitriiSukhomlinov** [comment]
@yalgaer yeah, now its good, please review

**Review by yalgaer — APPROVED** [review]

**Pull request merged**